src/Jackett.Common/Definitions/52pt.yml

```diff
@@ -40,10 +40,6 @@ settings:
     type: checkbox
     label: Search freeleech only
     default: false
-  - name: info_tpp
```

Comment:
why did you remove the ttp? all the nexus have this field in their control panel.

Reply:
oh! never mind. became obvious when I looked at the complete view rather than just the updates